packages/image_picker/image_picker_ios/ios/Classes/FLTPHPickerSaveImageToPathOperation.m

```diff
@@ -175,7 +175,7 @@ - (void)processImage:(UIImage *)localImage API_AVAILABLE(ios(14)) {
     NSString *savedPath =
         [FLTImagePickerPhotoAssetUtil saveImageWithPickerInfo:nil
                                                         image:localImage
-                                                 imageQuality:self.desiredImageQuality];
+                                                 imageQuality: @0.5];
```

Comment:
This isn't an actual fix for an issue, and it breaks functionally.

Reply:
Yes, I've put a comment below